charts/rollups-node/values.yaml

```diff
@@ -975,10 +975,13 @@ redis:
 postgresql:
   # -- Use bitnami postgreSQL pod.
   enabled: true
-  # -- Define an external Postgresql endpoint like postgres://host:pass@host:port/db
-  # REQUIRED if you disabled postgresql
+  # -- Define an existing secret name.
+  # REQUIRED if you disabled postgresql and don't want to define endpoint
+  SecretRef:
```

Comment:
it should be `secretRef`, but to keep consistency with other values on this same chart (see `dapp.mnemonic`), we should use:

```
postgresql.endpoint.value
postgresql.endpoint.secretRef
```

They should be mutually exclusive.